editor-components/modal/notification-modal1/notificationmodal1.tsx

```diff
@@ -0,0 +1,165 @@
+      },
+      value: [
+        {
+          type: "object",
+          key: "item",
+          displayer: "Item",
+          value: [
+            {
+              type: "string",
+              key: "buttonText",
+              displayer: "Button Text",
+              value: "Pay Now",
+            },
+            {
+              type: "page",
+              key: "link",
+              displayer: "Button Link",
+              value: "",
+            },
+          ],
+        },
+        {
+          type: "object",
+          key: "item",
+          displayer: "Item",
+          value: [
+            {
+              type: "string",
+              key: "buttonText",
+              displayer: "Button Text",
+              value: "Visit Page",
+            },
+            {
+              type: "page",
+              key: "link",
+              displayer: "Button Link",
+              value: "",
+            },
+          ],
+        },
+      ],
+    });
+  }
+
+  getName(): string {
+    return "NotificationModal1";
+  }
+
+  render() {
+    const backgroundImage = this.getPropValue("background_image");
+    const overlay = this.getPropValue("overlay");
+
+    return (
+      <div className={this.decorateCSS("container")}>
+        <div className={this.decorateCSS("max-content")}>
+          <div className={this.decorateCSS("content")}>
+            <div className={this.decorateCSS("image-container")}>
+              {backgroundImage && (
+                <>
+                  <img
+                    className={this.decorateCSS("image")}
+                    src={backgroundImage}
+                  />
+                  <h2 className={this.decorateCSS("first-header")}>
+                    {this.getPropValue("firstTitle")}
+                  </h2>
+                </>
+              )}
+              {overlay && (
+                <div className={this.decorateCSS("overlay")}></div>
+              )}
+              {!backgroundImage && overlay && (
+                <div className={this.decorateCSS("overlay2")}>
+                 <h2 className={this.decorateCSS("first-header")}>
+                  {this.getPropValue("firstTitle")}
+                </h2> 
+                </div>
+              )}
+            </div>
+            <div>
+              <h3 className={this.decorateCSS("second-header")}>
+                {this.getPropValue("secondTitle")}
+              </h3>
+              <p className={this.decorateCSS("notice-content")}>
+                {this.getPropValue("description")}
+              </p>
+            </div>
+            <div className={this.decorateCSS("button-background")}>
+              {this.castToObject<Button[]>("buttonprops").map((item: Button, index: number) => (
+                <ComposerModalClose key={index}>
+                  <a href={item.link} className={this.decorateCSS("button")}>
+                    {item.buttonText}
+                  </a>
+                </ComposerModalClose>
+              ))}
```

Comment:
if item.buttonText is empty, do not render it.

```suggestion
              {this.castToObject<Button[]>("buttonprops").map((item: Button, index: number) => {
                if(this.castToString(item.buttonText))
                  return (
                  <ComposerModalClose key={index}>
                    <a href={item.link} className={this.decorateCSS("button")}>
                      {item.buttonText}
                    </a>
                  </ComposerModalClose>
                )
              })}
```